bundles/org.openhab.binding.networkupstools/README.md

```diff
@@ -0,0 +1,127 @@
+| upsPower                   | Number:Power             | VA   | Current value of apparent power (Volt-Amps)                                        | yes           |
+| upsRealpower               | Number:Power             | W    | Current value of real power (Watts)                                                | no            |
+| upsStatus                  | String                   |      | Status of the UPS: OFF, OL,OB,LB,RB,OVER,TRIM,BOOST,CAL,BYPASS,NULL                | no            |
+| upsTemperature             | Number:Temperature       | °C   | UPS temperature (degrees C)                                                        | yes           |
+| upsTestResult              | String                   |      | Results of last self test (opaque string)                                          | yes           |
+| inputCurrent               | Number:ElectricCurrent   | A    | Input current (A)                                                                  | yes           |
+| inputCurrentStatus         | String                   |      | Status relative to the thresholds                                                  | yes           |
+| inputLoad                  | Number:Dimensionless     | %    | Load on (ePDU) input (percent of full)                                             | no            |
+| inputRealpower             | Number:Power             | W    | Current sum value of all (ePDU) phases real power (W)                              | yes           |
+| inputQuality               | String                   |      | Input power quality (*** opaque)                                                   | yes           |
+| inputTransferReason        | String                   |      | Reason for last transfer to battery (*** opaque)                                   | yes           |
+| inputVoltage               | Number:ElectricPotential | V    | Input voltage (V)                                                                  | yes           |
+| inputVoltageStatus         | String                   |      | Status relative to the thresholds                                                  | yes           |
+| batteryCharge              | Number:Dimensionless     | %    | Battery charge (percent)                                                           | no            |
+| batteryRuntime             | Number:Time              | s    | Battery runtime (seconds)                                                          | no            |
+| batteryVoltage             | Number:ElectricPotential | V    | Battery voltage (V)                                                                | yes           |
+
+### Dynamic Channels
+
+Because there is a lot of variation in ups features the binding supports dynamically adding channels for features not supported out-of-the-box.
+To get data from another nut variable the channel needs to configured.
```

Comment:
Add a sentence indicating that this can only be done through .things files.

I know you want to get this submitted without adding new features, but it would be nice if this binding had the ability to add Channels through the UI like the MQTT 2 Generic Things allows.

Reply:
That was actually what I was still missing. I wasn't fully aware of this feature. So I've updated the code to fully support this (I had the needed xml configuration already available.) So no need to update the documentation :smile:  and thanks for pointing this out.